appinventor/appengine/src/com/google/appinventor/server/storage/ObjectifyStorageIo.java

```diff
@@ -222,7 +222,7 @@ public void run(Objectify datastore) {
           user.setUserTosAccepted(userData.tosAccepted || !requireTos.get());
           user.setSessionId(userData.sessionid);
         }
-      });
+      }, false);
```

Comment:
It might be useful to put comments at the places where useTranscaction is false, noting this, and saying why transactions are not necessary in those cases.